cdk/lib/cdk-stack.ts

```diff
@@ -3,17 +3,43 @@ import { Construct } from "constructs";
 
 import * as s3 from "aws-cdk-lib/aws-s3";
 import * as s3deploy from "aws-cdk-lib/aws-s3-deployment";
+import * as lambda from "aws-cdk-lib/aws-lambda";
+
+const BUILD_DIR = "../build";
 
 export class CdkStack extends cdk.Stack {
```

Comment:
This wasn't the best choice of CDK stack name, but I can't be bothered now. I think I can come back and "change" it later by recreating the stack with the same underlying resources, but that will be quite annoying.